chunkflow/flow/cutout.py

```diff
@@ -42,8 +42,8 @@ def __call__(self, output_bbox):
                           progress=self.verbose,
                           mip=self.mip,
                           cache=False,
-                          green_threads=True)
-       
+                          green_threads=False)
```

Comment:
@ranlu why do you guys turn off green threads?

Reply:
Like I said in the commit message, for some reason the cloudvolume call hangs. I did not try harder to debug the exact cause, just modified the default.

Reply:
I think you might want to cherry pick some of the commit rather than merge the whole thing, the change I made are quite hackish and barely enough to run chunkflow in seuronbot. They need some work to make the function general enough I think.